### ISSUE TITLE
Crash when go to sms screen 

### ISSUE THREAD

**Issue by Paul75** [aj3423/SpamBlocker#115]
Hello when want to go to sms it crash.

```
android version: 31
app version: 1.20
java.lang.RuntimeException: java.lang.reflect.InvocationTargetException
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:596)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1211)
Caused by: java.lang.reflect.InvocationTargetException
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:586)
	... 1 more
Caused by: java.io.FileNotFoundException: content://com.android.contacts/contacts/6553/photo
	at android.database.DatabaseUtils.readExceptionWithFileNotFoundExceptionFromParcel(DatabaseUtils.java:151)
	at android.content.ContentProviderProxy.openTypedAssetFile(ContentProviderNative.java:780)
	at android.content.ContentResolver.openTypedAssetFileDescriptor(ContentResolver.java:2099)
	at android.content.ContentResolver.openAssetFileDescriptor(ContentResolver.java:1914)
	at android.content.ContentResolver.openInputStream(ContentResolver.java:1590)
	at android.graphics.ImageDecoder$ContentResolverSource.createImageDecoder(ImageDecoder.java:278)
	at android.graphics.ImageDecoder.decodeBitmapImpl(ImageDecoder.java:1870)
	at android.graphics.ImageDecoder.decodeBitmap(ImageDecoder.java:1969)
	at g2.h.d(Unknown Source:170)
	at q0.U.k(Unknown Source:1368)
	at q0.x.b(Unknown Source:62)
	at androidx.recyclerview.widget.LinearLayoutManager.W0(Unknown Source:0)
	at androidx.recyclerview.widget.LinearLayoutManager.K0(Unknown Source:48)
	at androidx.recyclerview.widget.LinearLayoutManager.d0(Unknown Source:800)
	at androidx.recyclerview.widget.RecyclerView.u(Unknown Source:73)
	at androidx.recyclerview.widget.RecyclerView.onMeasure(Unknown Source:63)
	at android.view.View.measure(View.java:26693)
	at android.widget.RelativeLayout.measureChildHorizontal(RelativeLayout.java:735)
	at android.widget.RelativeLayout.onMeasure(RelativeLayout.java:481)
	at android.view.View.measure(View.java:26693)
	at android.view.ViewGroup.measureChildWithMargins(ViewGroup.java:7305)
	at android.widget.LinearLayout.measureChildBeforeLayout(LinearLayout.java:1580)
	at android.widget.LinearLayout.measureHorizontal(LinearLayout.java:1211)
	at android.widget.LinearLayout.onMeasure(LinearLayout.java:730)
	at android.view.View.measure(View.java:26693)
	at android.view.ViewGroup.measureChildWithMargins(ViewGroup.java:7305)
	at android.widget.FrameLayout.onMeasure(FrameLayout.java:194)
	at android.view.View.measure(View.java:26693)
	at android.view.ViewGroup.measureChildWithMargins(ViewGroup.java:7305)
	at android.widget.FrameLayout.onMeasure(FrameLayout.java:194)
	at android.view.View.measure(View.java:26693)
	at A.f.b(Unknown Source:497)
	at x.e.V(Unknown Source:238)
	at x.e.U(Unknown Source:590)
	at P.m.m(Unknown Source:35)
	at androidx.constraintlayout.widget.ConstraintLayout.l(Unknown Source:1293)
	at androidx.constraintlayout.widget.ConstraintLayout.onMeasure(Unknown Source:1447)
	at android.view.View.measure(View.java:26693)
	at android.view.ViewGroup.measureChildWithMargins(ViewGroup.java:7305)
	at android.widget.FrameLayout.onMeasure(FrameLayout.java:194)
	at androidx.appcompat.widget.ContentFrameLayout.onMeasure(Unknown Source:159)
	at android.view.View.measure(View.java:26693)
	at android.view.ViewGroup.measureChildWithMargins(ViewGroup.java:7305)
	at android.widget.LinearLayout.measureChildBeforeLayout(LinearLayout.java:1580)
	at android.widget.LinearLayout.measureVertical(LinearLayout.java:849)
	at android.widget.LinearLayout.onMeasure(LinearLayout.java:728)
	at android.view.View.measure(View.java:26693)
	at android.view.ViewGroup.measureChildWithMargins(ViewGroup.java:7305)
	at android.widget.FrameLayout.onMeasure(FrameLayout.java:194)
	at android.view.View.measure(View.java:26693)
	at android.view.ViewGroup.measureChildWithMargins(ViewGroup.java:7305)
	at android.widget.LinearLayout.measureChildBeforeLayout(LinearLayout.java:1580)
	at android.widget.LinearLayout.measureVertical(LinearLayout.java:849)
	at android.widget.LinearLayout.onMeasure(LinearLayout.java:728)
	at android.view.View.measure(View.java:26693)
	at android.view.ViewGroup.measureChildWithMargins(ViewGroup.java:7305)
	at android.widget.FrameLayout.onMeasure(FrameLayout.java:194)
	at com.android.internal.policy.DecorView.onMeasure(DecorView.java:1219)
	at android.view.View.measure(View.java:26693)
	at android.view.ViewRootImpl.performMeasure(ViewRootImpl.java:4800)
	at android.view.ViewRootImpl.measureHierarchy(ViewRootImpl.java:3235)
	at android.view.ViewRootImpl.performTraversals(ViewRootImpl.java:3688)
	at android.view.ViewRootImpl.doTraversal(ViewRootImpl.java:2874)
	at android.view.ViewRootImpl$TraversalRunnable.run(ViewRootImpl.java:10701)
	at android.view.Choreographer$CallbackRecord.run(Choreographer.java:1451)
	at android.view.Choreographer.doCallbacks(Choreographer.java:1165)
	at android.view.Choreographer.doFrame(Choreographer.java:1054)
	at android.view.Choreographer$FrameDisplayEventReceiver.run(Choreographer.java:1417)
	at android.os.Handler.handleCallback(Handler.java:966)
	at android.os.Handler.dispatchMessage(Handler.java:110)
	at android.os.Looper.loopOnce(Looper.java:205)
	at android.os.Looper.loop(Looper.java:293)
	at android.app.ActivityThread.main(ActivityThread.java:9697)
	... 3 more

```

**Comment by aj3423** [comment]
It seems that some contact doesn't have a photo, the app is trying to get that photo and crashed. I wonder why no one has reported this until now.
I'll look into it tomorrow, a quick workaround would be setting a photo for that contact(s).

**Comment by aj3423** [comment]
Strange, I can't build it now, some dependency libraries were deprecated previously, but now they don't even exist.
 
I'm upgrading the entire UI and this will be fixed when it's done.

**Comment by Paul75** [comment]
hello,

It a simple contact .....

**Comment by aj3423** [comment]
Fixed in v2.0
